vsts/windows_csharp.cmd

```diff
@@ -4,24 +4,15 @@
 setlocal
 
 set build-root=%~dp0..
-rem // resolve to fully qualified path
+@rem resolve to fully qualified path
 for %%i in ("%build-root%") do set build-root=%%~fi
 
 cd %build-root%
 
-rem Build Debug + run UTs
-call build.cmd -clean -configuration Debug
-if errorlevel 1 goto :eof
-
+@rem Build Debug + run UTs and select E2E
 taskkill /F /IM Simulator.exe
 start /D .\vsts\TpmSimulator Simulator.exe
-
-rem Build Release + run E2E Tests
-rem If the build is Delay Signed, this requires strong name validation disabled for our key:
```

Comment:
we should retain these comments; for reference

Reply:
added back